src/casp/services/db/models/users.py

```diff
@@ -52,18 +53,14 @@ class UserActivity(db.Base):
     key="total_cells_processed",
     prop=column_property(
         User.cells_processed
-        + sa.select(sa.func.sum(UserActivity.cell_count))
-        .where((UserActivity.user_id == User.id) & (UserActivity.event == UserActivityEvent.SUCCEEDED))
-        .scalar_subquery()
+        + sa.select(sa.func.sum(UserActivity.cell_count)).where(UserActivity.user_id == User.id).scalar_subquery()
```

Comment:
This is gonna give incorrect counts because it will factor in failures and also double count everything because the `STARTED` events also include the cell count

Reply:
Oh whoops!  Not sure why that got changed